connectivity/lwipstack/include/lwipstack/LWIPStack.h

```diff
@@ -503,6 +503,15 @@ class LWIP : public OnboardNetworkStack, private mbed::NonCopyable<LWIP> {
     nsapi_size_or_error_t socket_recvfrom(nsapi_socket_t handle, SocketAddress *address,
                                           void *buffer, nsapi_size_t size) override;
 
+    // TODO: Add doc
+    nsapi_size_or_error_t socket_recvmsg(nsapi_socket_t handle, SocketAddress *address,
+                                         void *data, nsapi_size_t size,
```

Comment:
Can you add the doc as requested in the todo ?

Reply:
Done